demo/App.vue

```diff
@@ -155,9 +156,9 @@ export default {
     },
     handleSearchSelect(item) {
       if (['user', 'folder'].indexOf(item._modelType) >= 0) {
-        this.browserLocation = item;
+        this.browserLocation = { type: item._modelType, id: item._id };
```

Comment:
per below comment: this is the sort of translation I think it would be best to avoid.